sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -148,7 +151,6 @@ def __init__(self, init_config, instance_config, service_check_handler):
             self.log.error(
                 "Invalid ADODB provider string %s, defaulting to %s", self.adoprovider, self.default_adoprovider
             )
-            self.adoprovider = self.default_adoprovider
```

Comment:
Why remove this?

Reply:
its already being set here, no? https://github.com/DataDog/integrations-core/pull/12402/files#diff-a38a2a09e86a943d0704a9f2922133905324820865ff052eff46209ee989d6d9L146